python-sdk/tutorials/automl-with-azureml/ONNX_batch_model_generator_automl_for_images.py

```diff
@@ -0,0 +1,254 @@
+import os
+import onnx
+import torch
+import argparse
+from azureml.core import Experiment
+from azureml.core.workspace import Workspace
+from azureml.train.automl.run import AutoMLRun
+from torchvision.ops._register_onnx_ops import _onnx_opset_version
+from azureml.automl.dnn.vision.common.logging_utils import get_logger
+from azureml.automl.dnn.vision.common.model_export_utils import load_model
+from azureml.automl.dnn.vision.object_detection_yolo.models.common import Conv, Hardswish
+import azureml.automl.core.shared.constants as shared_constants
+from azureml.automl.dnn.vision.common.constants import ArtifactLiterals
+from azureml.automl.dnn.vision.object_detection.common.constants import ModelNames
+
+logger = get_logger(__name__)
+
+
+def export_onnx_model(model, dummy_input, input_names, output_names, dynamic_axes, file_path, device):
+    """
+    Export the pytorch model to onnx model file.
+
+    :param model: Model object for faster_rcnn or yolo or mask_rcnn
+    :type model: torchvision.models.detection
```

Comment:
str?

Reply:
model is of following type:
![image](https://user-images.githubusercontent.com/7171558/154566016-2cb089e9-d3b1-4d06-a539-95a927aec9f4.png)